examples/json_test/json_test.c

```diff
@@ -0,0 +1,91 @@
+/* Very basic example that just demonstrates we can run at all!
```

Comment:
(Unless this is from somewhere else, in which case weh ave to keep their copyright/license choice, or unless you'd really like to license it another way.)